src/webportal/src/app/user/fabric/user-profile.jsx

```diff
@@ -139,6 +146,39 @@ const UserProfile = () => {
     });
   });
 
+  // click `add public ssh keys button` -> open dialog
+  const onAddPublicKeys = useCallback(async sshPublicKeys => {
+    setSSHProcessing(true);
+    let updatedSSHPublickeys = [];
```

Comment:
Please validate here: the user cannot add ssh keys with duplicate titles. (Title should be unique for one user)

Reply:
done